src/promptflow/promptflow/_sdk/operations/_experiment_operations.py

```diff
@@ -151,6 +152,78 @@ def stop(self, experiment: Experiment, **kwargs) -> Experiment:
         ExperimentOrchestrator(self._client, experiment).stop()
         return self.get(experiment.name)
 
+    @monitor_operation(activity_name="pf.experiment.test", activity_type=ActivityType.PUBLICAPI)
+    def test(self, experiment: Experiment, inputs=None, environment_variables=None, **kwargs) -> Experiment:
+        """Test an experiment.
+
+        :param experiment: Experiment yaml file path.
+        :type experiment: Union[Path, str]
+        :param inputs: Input parameters for flow.
+        :type inputs: dict
+        :param environment_variables: Environment variables for flow.
+        :type environment_variables: dict
+        """
+        from .._load_functions import _load_experiment_template
+        from .._submitter.experiment_orchestrator import ExperimentOrchestrator
+
+        experiment_template = _load_experiment_template(experiment)
+        output_path = kwargs.get("output_path", None)
+        session = kwargs.get("session", None)
+        skip_flow = kwargs.get("skip_flow", None)
+        skip_flow_output = kwargs.get("skip_flow_output", None)
```

Comment:
I think maybe we add a single parameter here? like a context parameter
{
  "node_name" : {
       output: 
       run_id:
    }
}

Reply:
done